common/test/views/support/cleaner/AtomCleanerTest.scala

```diff
@@ -93,6 +96,36 @@ class AtomCleanerTest extends FlatSpec
     doc.getElementsByClass("youtube-media-atom__bottom-bar__duration").html() should be("0:36")
   }
 
+  "Youtube template" should "use active asset" in {
+    val atom = youTubeAtom.get.media.head.copy(
+      activeVersion = Some(2L),
+      assets = Seq(
+        youTubeAsset,
+        youTubeAsset.copy(id = "gyVuRflcEKM", version = 2),
+        youTubeAsset.copy(id = "QRplDNMsS4U", version = 3)
+      )
+    )
+
+    val html = views.html.fragments.atoms.media(media = atom, displayCaption = false, mediaWrapper = None)(TestRequest())
+    val doc = Jsoup.parse(html.toString())
+
+    doc.getElementsByClass("youtube-media-atom__iframe").attr("id") should be("youtube-gyVuRflcEKM")
+  }
+
+  "Youtube template" should "use latest asset if no active version" in {
+    val atom = youTubeAtom.get.media.head.copy(
+      assets = Seq(
+        youTubeAsset.copy(id = "gyVuRflcEKM", version = 2),
+        youTubeAsset.copy(id = "QRplDNMsS4U", version = 3)
+      )
+    )
+
+    val html = views.html.fragments.atoms.media(media = atom, displayCaption = false, mediaWrapper = None)(TestRequest())
+    val doc = Jsoup.parse(html.toString())
+
+    doc.getElementsByClass("youtube-media-atom__iframe").attr("id") should be(s"youtube-gyVuRflcEKM")
```

Comment:
I am not sure I understand. Is the latest assets the head one? if yes why it is version `2` compared to version `3`